components/support/nimbus-fml/src/backends/kotlin/gen_structs/mod.rs

```diff
@@ -118,6 +118,7 @@ impl<'a> FeatureManifestDeclaration<'a> {
                 "org.mozilla.experiments.nimbus.internal.FeatureManifestInterface".to_string(),
                 "org.mozilla.experiments.nimbus.FeaturesInterface".to_string(),
                 "org.json.JSONObject".to_string(),
+                "kotlin.collections.listOf".to_string(),
```

Comment:
I'm surprised this is necessary.

Reply:
it doesn't need this! I forgot to remove it 👍